package/kedro_viz/server.py

```diff
@@ -122,6 +114,15 @@ def pretty_name(name):
     return jsonify({"snapshots": [{"nodes": nodes, "edges": edges, "tags": tags}]})
 
 
+@app.route("/")
+@app.route("/<path:subpath>")
```

Comment:
`@app.route` is moved because we do not want to affect `@app.route("/logs/nodes.json")` and `@app.route("/api/nodes.json")`? If that's the case, when we wanted to add another `@app.route`, it might be a bit confusing whether future developers should add it before or after `@app.route("/<path:subpath>")`.

Reply:
do you have a concrete suggestion?
personally I feel this largely comes down to responsibility to have a general understanding of the tools you are using and to test things

Reply:
Okay, I think they will notice if they put a new decorator below as it won't get executed.

Reply:
Probably nice to leave a comment in the docstring like `only the static directory gets this treatment and all other file fetches are sent with max_age=0`, so people understand why `send_from_directory` was needed in the future.